pom.xml

```
@@ -67,17 +67,19 @@
         <jenkins.version>2.121.1</jenkins.version>
         <java.level>8</java.level>
         <no-test-jar>false</no-test-jar>
+        <useBeta>true</useBeta>
         <git-plugin.version>3.1.0</git-plugin.version>
         <workflow-support-plugin.version>3.3</workflow-support-plugin.version>
         <scm-api-plugin.version>2.2.6</scm-api-plugin.version>
         <groovy-cps.version>1.27</groovy-cps.version>
         <structs-plugin.version>1.17</structs-plugin.version>
```

Released version of workflow-step-api causes a RequireUpperBoundDeps error:
```suggestion
         <structs-plugin.version>1.18</structs-plugin.version>
```